### PR TITLE
HPCC-19822 Manually added ESP/Binding/FeatureAuth does not contain path

### PR DESCRIPTION
**Pull request by kenrowland**
Add path attribute to AuthenticateFeature element in the esp.xsd

Signed-off-by: Ken Rowland <kenneth.rowland@lexisnexisrisk.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [x] Existing deployed queries will not be broken
  - [x] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by kenrowland** [comment]
@rpastrana Please review.
Note that the spacing of the file was messed up and I reformatted it. The new lines are at line 165. All other lines are simply whitespace changes. To see just the changes, you can select the ignore white space in the diff settings.

File: initfiles/componentfiles/configxml/esp.xsd.in [hunk 1]
```diff
@@ -163,12 +163,21 @@
                                             </xs:appinfo>
                                         </xs:annotation>
                                     </xs:attribute>
+                                    <xs:attribute name="path" type="xs:string" use="optional" default="/">
+                                        <xs:annotation>
+                                            <xs:appinfo>
+                                                <tooltip>The logical path of a resource used for authentication.</tooltip>
```
Comment (rpastrana) [inline initfiles/componentfiles/configxml/esp.xsd.in]: these paths are use for Authorization, correct?

**Review by rpastrana — APPROVED** [review]
@kenrowland looks reasonable. Try to keep formatting changes separate.
The tooltip message might be slightly misleading.

File: initfiles/componentfiles/configxml/esp.xsd.in (hunk 1 above)
Comment (kenrowland) [reply initfiles/componentfiles/configxml/esp.xsd.in]: As far as I know.

**Comment by kenrowland** [comment]
@richardkchapman  please merge

**Pull request merged**